data/actions/lib/actions.lua

```diff
@@ -1,3 +1,20 @@
+local wildGrowth = { 1499, 11099 } -- wild growth destroyable by machete
+local jungleGrass = { -- grass destroyable by machete
+	[2782] = 2781,
+	[3985] = 3984,
+	[19433] = 19431
+}
+local groundIds = {354, 355} -- pick usable ground
```

Comment:
Add spaces around the other curly brackets (looks better) or remove them from line 1